tools/settingsLocal.py.test

```diff
@@ -0,0 +1,49 @@
+from os.path import join, dirname, abspath
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': join(dirname(dirname(abspath(__file__))), 'db.sqlite3'),
+    }
+}
+
+SECRET_KEY = "*=)2_8tuf5qym&6szbr%=xz)g8aw=w3z)ltc=+d)iy7v7grj#!"
```

Comment:
The Django secret key in test can just be set to "_" or something similar. Random keys like the one proposed may make other developers think you've leaked a key :)

Reply:
Yes, of course, it's juste a random key in this case.
good idea to use only "_" letters